proto/execution/server_enum.proto

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2019 Temporal Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+syntax = "proto3";
+
+package execution;
+
+option go_package = "github.com/temporalio/temporal/.gen/proto/execution";
+
+enum WorkflowExecutionState {
+    WorkflowExecutionState_Created = 0;
```

Comment:
so, the reason all of the values are prefixed with `WorkflowExecutionState_` is that otherwise, some of the values conflict with the values in 

https://github.com/temporalio/temporal-proto/blob/bc281076245f1d4eced66af2fd29875e127b63c3/execution/enum.proto#L31-L32

(which are merged into the same package via Makefile magic).

I think this is probably acceptable, but not ideal, so if anyone has other ideas, please let me know!

Reply:
We only want to prefix the unknown values, so we can remove the prefixes here. Can follow up later when doing the Unspecified changes though.

enum WorkflowExecutionState {
   WorkflowExecutionState_Unspecified = 0;
   Created = 1;
}

Reply:
Yes, I think this is the general approach we converged on (aka "Option 1";), for dealing with `Unspecified` / `0` option (which I am not addressing here).

Separate from that, in this case, the naming conflict involves `Created` and `Running` defined here vs https://github.com/temporalio/temporal-proto/blob/bc281076245f1d4eced66af2fd29875e127b63c3/execution/enum.proto#L31-L32

Reply:
I don't like this. I see following options:
1. Rename `Running` and `Completed`. Like `InProgess` and `Finished`. For example `Success` is a good for `WorkflowExecutionStatus.Completed`. Having different names will also help to differentioate them from code.
2. Move one of them to `common`. I think `WorkflowExecutionState` is a better candidate.

Reply:
per our chat, wondering if you have a preference for a specific change here (or a preference for no change;).